lighthouse-cli/cli-flags.js

```diff
@@ -70,114 +64,218 @@ function getFlags(manualArgv) {
        *
        * @see https://github.com/yargs/yargs/blob/a6e67f15a61558d0ba28bfe53385332f0ce5d431/docs/api.md#config
        */
-      .config('cli-flags-path')
+      .option('cli-flags-path', {
+        config: true,
+        describe: 'The path to a JSON file that contains the desired CLI flags to apply. Flags specified at the command line will still override the file-based ones.',
+      })
 
-      // List of options
+      // Logging
```

Comment:
the slightly weird order of options in here is due to our currently weird order of options. This `cli-flags` produces *exactly* the same `--help` order except `--version` and `--help` are switched, and there are more annotated `[default: false]` at the end of lines.

Happy to reorder.